pkg/operator/sync.go

```diff
@@ -30,6 +31,10 @@ const (
 	machineExposeMetricsPort            = 8441
 	machineSetExposeMetricsPort         = 8442
 	machineHealthCheckExposeMetricsPort = 8444
+	defaultMachineHealthPort            = 9440
+	defaultMachineSetHealthPort         = 9441
+	defaultMachineHealthCheckHealthPort = 9442
```

Comment:
Might be being a bit pedantic, but would it be a pain to make these that same as the metrics ports but +1000 for consistency? WDYT?
```suggestion
	defaultMachineHealthPort            = 9441
	defaultMachineSetHealthPort         = 9442
	defaultMachineHealthCheckHealthPort = 9444
```

Reply:
It will be something breaking PRs across 5 repos 😁

Reply:
Oh really? 😓  We should be using constants for this really, but I'll let that be a future improvement